WooCommerce/Classes/Styles/Style.swift

```diff
@@ -176,6 +184,9 @@ private extension DefaultStyle {
 
         static let goldStarColor         = UIColor(red: 238.0/255.0, green: 180.0/255.0, blue: 34.0/255.0, alpha: 1.0)
         static let grayStarColor         = UIColor(red: 89.0/255.0, green: 89.0/255.0, blue: 89.0/255.0, alpha: 1.0)
+
+        // Muriel theme in https://color-studio.blog/
+        static let red50                 = UIColor(red: 0.84, green: 0.21, blue: 0.22, alpha: 1)
```

Comment:
Most, if not all, of the other Handbook colors make some reference to where they are used or what justifies them..., would it make sense to name this one murielRed50, or wooRedHighlight, or something much better than any of my suggestions? 😛